experimenter/experimenter/theme/static_src/src/styles.css

```diff
@@ -1,3 +1,63 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+
+[x-cloak] { 
+    display: none !important;
+}
+
+a[role=tab] {
+    border-bottom: 1px solid transparent;
+    transition: all var(--transition);
+    min-width: 120px;
+    text-align: center;
+    user-select: none;
+    width: 100%;
+    padding: 4px;
+    color: #938f8f;
+}
+a[role=tab].active {
+    color: #242D4Eff;
+    border-radius: 0;
+}
+a[role=tab]:focus {
+    background: transparent;
+}
+div.grid div {
+    min-height: 200px;
+    animation-duration: 0.45s;
+    -webkit-animation-duration: 0.45s;
+    animation-fill-mode: both;
+    -webkit-animation-fill-mode: both;
+    -webkit-animation-name: slideIn;
+    animation-name: slideIn;
+}
+@keyframes slideIn {
+    0% {
+        transform: translateY(1rem);
+        opacity: 0;
+    }
+    100% {
+        transform:translateY(0rem);
+        opacity: 1;
+    }
+    0% {
+        transform: translateY(1rem);
+        opacity: 0;
+    }
+}
+@-webkit-keyframes slideIn {
+    0% {
+        -webkit-transform: transform;
+        -webkit-opacity: 0;
+    }
+    100% {
+        -webkit-transform: translateY(0);
+        -webkit-opacity: 1;
+    }
+    0% {
+        -webkit-transform: translateY(1rem);
+        -webkit-opacity: 0;
+    }
+}
```

Comment:
This does not need prefixing. see https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes

Reply:
Oh, I took it from a example code. I'll update this